packages/web/src/pages/upload-page/fields/AccessAndSaleField.tsx

```diff
@@ -594,11 +599,12 @@ export const AccessAndSaleField = (props: AccessAndSaleFieldProps) => {
         messages.fieldVisibility
       ) as Array<keyof FieldVisibility>
 
-      const fieldVisibilityLabels = fieldVisibility
-        ? fieldVisibilityKeys
-            .filter((visibilityKey) => fieldVisibility[visibilityKey])
-            .map((visibilityKey) => messages.fieldVisibility[visibilityKey])
-        : []
+      const fieldVisibilityLabels =
+        fieldVisibility && !isAlbum
```

Comment:
not sure i understand this - seems like it's saying if it's not an album upload, we won't have any field visibility options at all?
also will this work for playlist upload?

Reply:
so this is logic that displays the current values for the inner form
![image](https://github.com/AudiusProject/audius-protocol/assets/2358254/f75eb88b-641b-42ac-bac4-56bc353be1c3)

For albums, we don't allow you to set the field visibility, so it's confusing to display the preview value for it. For playlists, we don't show the `AccessAndSaleField` at all since you can only upload public playlists currently.